ci_framework/roles/hive/tasks/openstack_delete.yml

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Please rename this file to `openstack_unclaim.yml`... as we would have `openstack_delete.yml` that would remove the cluster pool (in the future)